src/metabase/analytics/stats.clj

```diff
@@ -612,27 +612,42 @@
   (t/minus (t/offset-date-time) (t/days 1)))
 
 (defn- ->snowplow-grouped-metric-info []
-  {:query_executions (merge
-                      {"sdk_embed" 0 "interactive_embed" 0 "static_embed" 0 "public_link" 0 "internal" 0}
-                      (-> categorize-query-execution
-                          (group-by
-                           (t2/select [:model/QueryExecution :embedding_client :executor_id]))
-                          (update-vals count)))})
-
-(defn- snowplow-grouped-metrics [{query-executions :query_executions :as _snowplow-grouped-metric-info}]
-  (->> [{:name :query_executions_by_source
-         :values (mapv (fn [qe-group]
-                         {:group qe-group :value (get query-executions qe-group)})
-                       ["interactive_embed" "internal" "public_link" "sdk_embed" "static_embed"])
-         :tags ["embedding"]}]
-       (walk/postwalk (fn [x] (if (keyword? x) (-> x u/->snake_case_en name) x)))))
+  (let [qe (t2/select [:model/QueryExecution :embedding_client :executor_id :started_at])
+        one-day-ago (->one-day-ago)
+        ;; reuse the query data:
+        qe-24h (filter (fn [{started-at :started_at}] (->> started-at (t/after? one-day-ago))) qe)]
```

Comment:
`t/after?` always seems to have the wrong order of args, but I might have figured it out, if you treat it like an infix operator: then, I think, it reads much better.

Reply:
just to keep my sanity:

``` clojure
(defonce earlier (java.time.Instant/now))
;; wait a sec
(def later (java.time.Instant/now))

(t/after? later earlier)
;; => true

;; and:

(->> earlier (t/after? later))
;; => true
```